pkg/kapp/app/recorded_app.go

```diff
@@ -39,8 +41,10 @@ var _ App = &RecordedApp{}
 func (a *RecordedApp) Name() string      { return a.name }
 func (a *RecordedApp) Namespace() string { return a.nsName }
 
+func (a *RecordedApp) CreationTimestamp() *time.Time { return &a.creationTimestamp }
+
 func (a *RecordedApp) Description() string {
-	return fmt.Sprintf("app '%s' namespace: %s", a.name, a.nsName)
+	return fmt.Sprintf("app '%s' namespace: %s", a.Name(), a.Namespace())
```

Comment:
(i seemed to have lost a comment i thought i made...) lets use a.name and a.namespace within this class. i dont see a reason why we need to use public methods in this PR (will make it harder for other folks to rebase if they changed this file in active PRs). additionally generally within classes we do not use public methods to reference necessary info -- if you starting to do so, if you find yourself using public methods in other public methods, this might be an indication that these are 2 different classes shoved into one.

Reply:
My motivation behind this is as I have got comments saying that can we have something like `RecordedAppMeta` struct inside `RecordedApp` struct to segregate the meta fields so I feel like we should start using methods so that if we go that way we just have to change the methods in future. 

But as we are not adding something like `RecordedAppMeta` and rebase can be an issue I can revert this.